qtconsole/jupyter_widget.py

```diff
@@ -292,22 +291,30 @@ def _started_channels(self):
     def _process_execute_error(self, msg):
         """Handle an execute_error message"""
         content = msg['content']
-        traceback = '\n'.join(content['traceback']) + '\n'
-        if False:
-            # FIXME: For now, tracebacks come as plain text, so we can't use
-            # the html renderer yet.  Once we refactor ultratb to produce
-            # properly styled tracebacks, this branch should be the default
-            traceback = traceback.replace(' ', '&nbsp;')
-            traceback = traceback.replace('\n', '<br/>')
-
-            ename = content['ename']
-            ename_styled = '<span class="error">%s</span>' % ename
-            traceback = traceback.replace(ename, ename_styled)
-
-            self._append_html(traceback)
+
+        if content['ename'] == 'KeyboardInterrupt':
+            # If the user interrupted execution by pressing CTRL-C, only
+            # display a simple message.
+            self._append_plain_text("Execution aborted\n")
```

Comment:
I like this idea a lot. It'd prevent displaying long tracebacks that don't mean much to the user.

@takluyver, what do you think about it?